hub/auto/load.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import pathlib
+from typing import Callable, Dict, List, Union
+from hub.util.exceptions import HubAutoUnsupportedFileExtensionError
+
+from PIL import Image
+
+
+IMAGE_SUFFIXES: List[str] = [".jpeg", ".jpg", ".png"]
+SUPPORTED_SUFFIXES: List[str] = IMAGE_SUFFIXES
+
+
+def _load_image(image_path: str) -> np.ndarray:
+    img = Image.open(image_path)
+    return np.array(img)
+
+
+def load(path: Union[str, pathlib.Path], symbolic=False) -> Union[Callable, np.ndarray]:
+    # TODO: write load tests
+    path = pathlib.Path(path)
+
+    suffix = path.suffix.lower()
+    if suffix in IMAGE_SUFFIXES:
+        if symbolic:
```

Comment:
symbolic here would be helpful for when loading a sample that doesn't fit into memory (this function `hub.load` can be our super-power :-))

Reply:
also `symbolic` can be used for lazy uploading if we do this in the future